com.unity.formats.alembic/Editor/Importer/AlembicStreamPlayerEditor.cs

```diff
@@ -295,5 +295,16 @@ void UndoRedoPerformed()
 
             loadSucceded = (target as AlembicStreamPlayer).ReloadStream();
         }
+
+        [MenuItem("CONTEXT/AlembicStreamPlayer/Reset"), ]
+        static void ResetPreventer()
```

Comment:
Not sure I'm reading this right. It's a context menu that's always disabled and does nothing?

Reply:
All good.